target/linux/ath79/dts/qca9558_comfast_cf-wr650ac.dtsi

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wps {
+			label = "comfast:blue:wps";
```

Comment:
Please use the naming scheme for most of the ath79 LEDs. `<boardname>:<color>:<function>`

Therefore, move the LED definitions to the devices specific DTS.

Reply:
Can you clarify exactly what is wrong here ?

Reply:
applied the same logic as in Yuncore 830 PR